RGLServerPlugin/src/LidarPatternLoader.cc

```diff
@@ -25,32 +26,41 @@ namespace fs = std::filesystem;
 namespace rgl
 {
 
-std::map<std::string, std::string> LidarPatternLoader::presetNameToFilename = {
-    {"Alpha Prime", "VelodyneVLS128.mat3x4f"},
-    {"Puck", "VelodyneVLP16.mat3x4f"},
-    {"Ultra Puck", "VelodyneVLP32C.mat3x4f"},
-    {"OS1 64", "OusterOS1_64.mat3x4f"},
-    {"Pandar64", "HesaiPandarQT64.mat3x4f"},
-    {"Pandar40P", "HesaiPandar40P.mat3x4f"}
+// map preset name to pair of filename and sample size (for alternating patterns)
+// set sample size to 0 to indicate non-alternating patterns
+std::map<std::string, std::pair<std::string, std::size_t>> LidarPatternLoader::presetNameToFilename = {
+    {"Alpha Prime", {"VelodyneVLS128.mat3x4f", 0}},
+    {"Puck", {"VelodyneVLP16.mat3x4f", 0}},
+    {"Ultra Puck", {"VelodyneVLP32C.mat3x4f", 0}},
+    {"OS1 64", {"OusterOS1_64.mat3x4f", 0}},
+    {"Pandar64", {"HesaiPandarQT64.mat3x4f", 0}},
+    {"Pandar40P", {"HesaiPandar40P.mat3x4f", 0}},
+    {"Livox Avia", {"LivoxAvia.mat3x4f", 24000}},
+    {"Livox Horizon", {"LivoxHorizon.mat3x4f", 24000}},
+    {"Livox Mid40", {"LivoxMid40.mat3x4f", 10000}},
+    {"Livox Mid70", {"LivoxMid70.mat3x4f", 10000}},
+    {"Livox Mid360", {"LivoxMid360.mat3x4f", 20000}},
+    {"Livox Tele15", {"LivoxTele15.mat3x4f", 24000}},
```

Comment:
Could you provide update rate of the LiDAR this setup is prepared for? It is worth noting in the README. 

(We should also specify it for our existing presets)